src/extensions/default/InlineColorEditor/ColorProperties.json

```diff
@@ -0,0 +1,13 @@
+{
+    "background-color":            {"values": ["inherit"], "type": "color"},
+    "border-color":                {"values": ["inherit"], "type": "color"},
+    "border-bottom-color":         {"values": ["inherit"], "type": "color"},
+    "border-left-color":           {"values": ["inherit"], "type": "color"},
+    "border-right-color":          {"values": ["inherit"], "type": "color"},
+    "border-top-color":            {"values": ["inherit"], "type": "color"},
+    "color":                       {"values": ["inherit"], "type": "color"},
+    "column-rule-color":           {"values": [], "type": "color"},
+    "outline-color":               {"values": ["invert", "inherit"], "type": "color"},
+    "text-decoration-color":       {"values": [], "type": "color"},
+    "text-emphasis-color":         {"values": [], "type": "color"}
```

Comment:
I don't think there's any value having `"type": "color"` on every one of these: because they all have it, you might as well get rid of it everywhere, and remove the check below for `type = "color"`.